File: easiness-core/src/main/java/com/easiness/core/entities/PlaceEntity.java
package com.easiness.core.entities;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "place")
public class PlaceEntity extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    @Column(name = "id", nullable = false, updatable = false)
    Long id;

    @Column(name = "name", unique = true)
    String name;

    @Column(name = "address")
    String address;

}

File: easiness-ui/app/server/entity/place.entity.ts
import { Entity, PrimaryGeneratedColumn, Column } from "typeorm"
import { Base } from "./base.entity"


@Entity({ name: 'place' })
export class Place extends Base {
    @PrimaryGeneratedColumn()
    id: number

    @Column({ unique: true })
    name: string

    @Column()
    address: string
}